common/djangoapps/third_party_auth/pipeline.py

```diff
@@ -143,16 +142,11 @@ def B(*args, **kwargs):
     AUTH_ENTRY_LOGIN_2: '/account/login/',
     AUTH_ENTRY_REGISTER_2: '/account/register/',
 
-    # If linking/unlinking an account from the new student profile
-    # page, redirect to the profile page.  Only used if
-    # `FEATURES['ENABLE_NEW_DASHBOARD']` is true.
-    AUTH_ENTRY_PROFILE: '/profile/',
```

Comment:
@wedaly can you explain this third_party_auth pipeline code and why it references the "new student profile page"? @ormsbee is a bit nervous that someone might be using this code (though ENABLE_NEW_DASHBOARD has never been True in edx production.